peekingduck/pipeline/nodes/input/recorded.py

```diff
@@ -27,7 +28,7 @@ def __init__(self, config: Dict[str, Any]) -> None:
         super().__init__(config, node_path=__name__)
         self._allowed_extensions = ["jpg", "jpeg", "png", "mp4", "avi", "mov"]
```

Comment:
Add ".mkv" extension as one of our clients recently provided videos in this format.

Reply:
added.